src/test/java/org/gridsuite/shortcircuit/server/repositories/FaultResultRepositoryTest.java

```diff
@@ -129,6 +139,66 @@ void faultResultFilterWithPageableTest(ShortCircuitAnalysisResultEntity resultEn
         //Test with pageable containing a sort by nbLimitViolations and since some values are equals we except the result to be sorted by nbLimitViolations first and then by uuid
         faultPage = shortCircuitAnalysisResultRepository.findFaultResultsPage(resultEntity, resourceFilters, PageRequest.of(0, 3, Sort.by(new Sort.Order(Sort.Direction.ASC, "nbLimitViolations"))), FaultResultsMode.BASIC);
         assertFaultEqualsInOrder(faultPage, Comparator.comparing(FaultResultEntity::getNbLimitViolations).thenComparing(o -> o.getFaultResultUuid().toString()));
+
+        // Test with pageable containing :
+        // - a descending primary sort by current
+        // - a descending secondary sort by connectableId
+        faultPage = shortCircuitAnalysisResultRepository.findFaultResultsPage(
+                resultEntity,
+                resourceFilters,
+                PageRequest.of(0, 3, Sort.by(
+                        new Sort.Order(Sort.Direction.DESC, "current"),
+                        new Sort.Order(Sort.Direction.DESC, "connectableId"))),
+                FaultResultsMode.FULL);
+        assertFeedersEqualsInOrder(faultPage,
+                Comparator.comparing(FaultResultEntity::getCurrent).reversed(),
+                Comparator.comparing(FeederResultEntity::getConnectableId).reversed());
+    }
+
+    @ParameterizedTest(name = "[{index}] Using the filter(s) {1} should return the given entities")
```

Comment:
There is no point to use `ParameterizedTest` if you have only one input

Reply:
I agree, but in my opinion the fix should be adding more test cases rather than not using ParameterizedTest

Reply:
Indeed. Added a few more input, including CONTAINS, START_WITH, one without any filter and one with a double filter.